source/slang/glsl.meta.slang

```diff
@@ -7263,3 +7263,704 @@ __glsl_version(430) [require(glsl)]
     }
     }
 }
+
+/// Section 8.14. Fragment Processing Functions
+
+[require(hlsl, sm_4_0)]
+__glsl_version(430) [require(glsl)]
+__spirv_version(1.0) [require(spirv)]
```

Comment:
need to require `fragment` capability here, since this is only available for fragment shaders?